app/views/tenants/projects/index.html.erb

```diff
@@ -12,6 +12,15 @@
         <% project.categories.each do |category| %>
           <div class="project-category"><%= category.name %></div>
         <% end %>
+        <%= form_for :pending_loan, url: carts_path do |f| %>
+          <%= f.hidden_field :project_id, value: project.id %>
+          <%= f.select(:loan_amount) do %>
+            <% [["$25", 25], ["$50", 50], ["$100", 100], ["$500", 500]].each do |amount| %>
```

Comment:
Same thing as above, not sure about going to multiple amounts this early in the game :smile: